File: src/main/java/com/tva/mk/req/BaseReq.java
package com.tva.mk.req;

import com.fasterxml.jackson.annotation.JsonProperty;

public class BaseReq {
	// region -- Fields --

	@JsonProperty(value = "keyword")
	private String keyword;

	@JsonProperty(value = "isOptional")
	private Boolean isOptional;

	// end

	// region -- Get set --

	public String getKeyword() {
		return keyword;
	}

	public void setKeyword(String keyword) {
		this.keyword = keyword;
	}

	public Boolean getIsOptional() {
		return isOptional;
	}

	public void setIsOptional(Boolean isOptional) {
		this.isOptional = isOptional;
	}

	// end

	// region -- Methods --

	public BaseReq() {
	}

	public BaseReq(String keyword) {
		this.keyword = keyword;
	}

	// end
}

File: dotnet/MoneyKeeper.Main/MoneyKeeper.Main/Req/BaseReq.cs
using Newtonsoft.Json;

namespace MoneyKeeper.Main.Req
{
    /// <summary>
    /// Base request
    /// </summary>
    public class BaseReq
    {
        #region -- Properties --

        [JsonProperty("keyword")]

        public object Keyword { get; set; }

        [JsonProperty("isOptional")]
        public bool? IsOptional { get; set; }

        #endregion
    }
}